<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* General style for all QComboBox */
QComboBox {
    border: 1px solid #ced4da;
    border-radius: 4px;
    padding-left: 10px;
}

/* Style for the drop-down area */
QComboBox::drop-down {
    border: 0px;
}

/* Style for the drop-down arrow */
QComboBox::down-arrow {
    image: url(C:/Users/delta/OneDrive/Desktop/sampling/.idea/25623.png);
    width: 12px;
    height: 12px;
    margin-right: 15px;
}
QPushButton {

border-radius: 8px;
padding: 1px 5px;
min-width: 10px;
min-height:20px;
color: #fff;
background-color: #212529
}
QPushButton::hover{
background-color: #1c1f23;
}
QPushButton:pressed{
background-color: #a0a2a4;
}
QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 2px; /* Adjust the height instead of width for horizontal slider */
}

QSlider::handle:horizontal {
   /* background: #eb583b;*/
/*background:#9c1616;*/
background:#ab1818;
    width: 10px; /* Use width instead of height for horizontal slider */
    margin: -6px -1px; /* Adjust margins for horizontal alignment */
    border-radius: 5px;
    border: 0px solid #2a2a2a;
    height: 10px; /* Adjust height of handle for horizontal slider */
}

QSlider::add-page:horizontal {
/*background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #e3ac76, stop:1 #e3ac76);*/
/*background:rgba(76, 175, 80, 0.3);*/
/* This color will appear on the right side of the slider */
}

QSlider::sub-page:horizontal {
/*background: #eb583b;*/ /* Adjust gradient direction for horizontal slider */
/*background: #9c1616; *//*red-dark*/
background:#ab1818;
}

QSlider::handle:horizontal:hover {
    background: red;
    border-color: red;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::tab-bar:top {
    top: 1px;
}

QTabWidget::tab-bar:bottom {
    bottom: 1px;
}

QTabWidget::tab-bar:left {
    right: 1px;
}

QTabWidget::tab-bar:right {
    left: 1px;
}

QTabBar::tab {
    border: 1px solid black;
}

QTabBar::tab:selected {
    background: white;
}

QTabBar::tab:!selected {
    background: silver;
}

QTabBar::tab:!selected:hover {
    background: #999;
}

QTabBar::tab:top:!selected {
    margin-top: 3px;
}

QTabBar::tab:bottom:!selected {
    margin-bottom: 3px;
}

QTabBar::tab:top, QTabBar::tab:bottom {
    min-width: 8ex;
    margin-right: -1px;
    padding: 5px 10px 5px 10px;
}

QTabBar::tab:top:selected {
    border-bottom-color: none;
}

QTabBar::tab:bottom:selected {
    border-top-color: none;
}

QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one {
    margin-right: 0;
}

QTabBar::tab:left:!selected {
    margin-right: 3px;
}

QTabBar::tab:right:!selected {
    margin-left: 3px;
}

QTabBar::tab:left, QTabBar::tab:right {
    min-height: 8ex;
    margin-bottom: -1px;
    padding: 10px 5px 10px 5px;
}

QTabBar::tab:left:selected {
    border-left-color: none;
}

QTabBar::tab:right:selected {
    border-right-color: none;
}</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Siamese">
       <attribute name="title">
        <string>Siamese</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_11">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="first_img_original">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>450</width>
               <height>450</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgba(76, 175, 80, 0.3);
color: white</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="add_img1_btn">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>140</y>
               <width>141</width>
               <height>131</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:none;

  </string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_12">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="second_img_original">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>450</width>
               <height>450</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgba(76, 175, 80, 0.3);
color: white</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="add_img2_btn">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>140</y>
               <width>141</width>
               <height>131</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:none;</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>800</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="Match_button_1">
             <property name="geometry">
              <rect>
               <x>340</x>
               <y>20</y>
               <width>131</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Match</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CNN">
       <attribute name="title">
        <string>CNN</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_13">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="first_img_original_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>450</width>
               <height>450</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgba(76, 175, 80, 0.3);
color: white</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="add_img1_btn_3">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>140</y>
               <width>141</width>
               <height>131</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:none;</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>800</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>150</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="submit_cnn">
             <property name="geometry">
              <rect>
               <x>320</x>
               <y>10</y>
               <width>131</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Match</string>
             </property>
            </widget>
            <widget class="QProgressBar" name="progressBar">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>80</y>
               <width>451</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Identification">
       <attribute name="title">
        <string>Identification</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFrame" name="frame_15">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>450</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="first_img_original_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>450</width>
               <height>450</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>450</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: rgba(76, 175, 80, 0.3);
color: white</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="add_img1_btn_4">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>140</y>
               <width>141</width>
               <height>131</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>100</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:none;</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="minimumSize">
             <size>
              <width>800</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_3">
             <property name="geometry">
              <rect>
               <x>330</x>
               <y>10</y>
               <width>131</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Identify</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
